modules/Microsoft.Insights/webTests/.test/min/deploy.test.bicep

```diff
@@ -0,0 +1,78 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.insights.webtests-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'wtmin'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-nestedDependencies'
+  params: {
+    appInsightName: 'dep-<<namePrefix>>-appi-${serviceShort}'
+    logAnalyticsWorkspaceName: 'dep-<<namePrefix>>-law-${serviceShort}'
+    location: location
+  }
+}
```

Comment:
are these dependencies actually necessary? I see only the AppInsightsResourceId is used - but for a tag.

Reply:
Application insights resource required log analytics workspace

Reply:
I'd share the same doubt as @AlexanderSehr. I'm not experienced with this resource type, but I don't see the app insight resource referenced anywhere within the module, so I'm not sure how they are related. Could you help us elaborating a bit more?

Reply:
It turns out the tag actually must be implemented like @shaban-khan did. The hidden-link is a mandatory tag, as weird as this may sound. We could at most argue to introduce a new parameter of type array that is more 'explicit' and is unioned into the tags. However, this would diverge from the RP implementation